javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,158 @@
+let savedName = " ";
+const toDoList = [];
+
+/**
+ * function to get Reply from voice assistant
+ * using input as command
+ * @param {*} command
+ */
+function getReply(command) {
+  if (typeof command != "string" || typeof command == undefined) {
+    return "Invalid input";
+  }
+  const input = command.toLowerCase();
+  let activity;
+
+  if (input.includes("hello my name is")) {
+    let token = input.split(" ");
+    const username = token[token.length - 1];
+    if (savedName === " ") {
```

Comment:
You could also define `savedName` as an empty string and use `.length` to check. 
Also remember [truthy values](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) 
```js
let name = ""

if (name) { // this will evaluate to false because it's empty
    //
}
```